assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -196,6 +198,14 @@
 	
 	private static final Boolean allowReviewService = ServerConfigurationService.getBoolean("assignment.useContentReview", false);
 	private static final Boolean allowPeerAssessment = ServerConfigurationService.getBoolean("assignment.usePeerAssessment", true);
+	private static Boolean allowLTIReviewService = false;
+	private static Boolean isDirectAccess = false;
```

Comment:
These static variables are referenced across multiple user sessions, so this is likely to cause concurrency issues. For instance, two users working in sites with opposite ContentReviewSiteAdvisor settings will flip these on eachother.

Now, AssignmentAction is a servlet-class, and so I believe it's somewhat of a singleton, so simply removing the static won't fix this; the ContentReviewSiteAdvisor will have to be consulted for every usage these.